CONTRIBUTING.md

```diff
@@ -243,18 +243,82 @@ If in doubt, you can always ask for guidance in the Pull Request or on
+In order to get landed, a Pull Request needs to be reviewed and
+[approved](#getting-approvals-for-your-pull-request) by
+at least one Node.js Collaborator. If this Pull Request touches
+more than the documentation, then it also needs an all-green
+[CI (Continuous Integration) test run](#ci-testing). After that,
+as long as there is no objections from a Collaborator, this
+Pull Request can be merged. If you find your Pull Request waiting
+longer than you expect, see
+[notes about the waiting time](#waiting-until-the-pull-request-gets-landed).
+
+When the Pull Request is landed, the person who land this will post
+a comment to the Pull Request page mentioning the commit(s) in which
+it is landed. GitHub often shows the Pull Request as `Closed` at this
+point, but don't worry. If you look at the branch you raised your
+Pull Request against (probably `master`), you should see a commit with
+your name on it. Congratulations and thanks for your contribution!
+
+## Additional Notes
+
+### Commit Squashing
+
+When the commits in your Pull Request get landed, they will be squashed
+into one commit per logical change, with metadata added to the commit
+message (including links to the Pull Request, links to relevant issues,
+and the names of the reviewers). As long as you don't delete
+your fork branch, the commit history of your Pull Request will stay
+intact on the Pull Request page.
+
+For the size of "one logical change",
+[0b5191f](https://github.com/nodejs/node/commit/0b5191f15d0f311c804d542b67e2e922d98834f8)
+can be a good example. It touches the implementation, the documentation,
+and the tests, but is still one logical change.
+
+### Getting Approvals for Your Pull Request
+
+The approval can be made either by saying LGTM, which stands for
+"Looks Good To Me", or by using GitHub's Approve button. 
+GitHub's Pull Request review feature is used during the process.
+For more information, check out
+[the video tutorial](https://www.youtube.com/watch?v=HW0RPaJqm4g)
+or [the official documentation](https://help.github.com/articles/reviewing-changes-in-pull-requests/).
+
+After you push new changes to your branch, you need to get the
+approval for these new changes again, even if GitHub shows "Approved"
+because the reviwers have hit the buttons before.
+
+### CI Testing
+
+If your Pull Request contains executable code, it needs to be tested
+to make sure the code works on the platforms that Node.js
+supports. This is done by running the code through the CI system.
+
+Only a Collaborator can request a CI run. Usually one of them will do it
+for you as approvals for the Pull Request come in.
+If not, you can ask a Collaborator to request a CI run.
+
+### Waiting Until the Pull Request Gets Landed
+
+If there are non-trivial changes in this Pull Request, after it
+gets approved and passes the CI, it still needs to wait
+for at least another 48 hours (72 hours on a weekend)
+so that more input can be made before it gets landed.
```

Comment:
I'm probably being pedantic here, but even if it is a trivial change, unless collaborators specifically say that there's no need to wait the full time as it's trivial, it will by default still have the 48/72 hour wait.

Maybe:
```diff
- If there are non-trivial changes in this Pull Request, after it
- gets approved and passes the CI, it still needs to wait
- for at least another 48 hours (72 hours on a weekend)
- so that more input can be made before it gets landed.

+ After the Pull Request gets approved and passes the CI,
+ it still needs to wait for at least another 48 hours (72
+ hours on a weekend) so that everyone has a chance to 
+ weigh in. If the changes are trivial, collaborators may 
+ decide it doesn't need to wait.
```